src/Models/T1_relaxometry/vfa_t1.m

```diff
@@ -99,10 +99,17 @@
             % T1 and M0
             flipAngles = (obj.Prot.VFAData.Mat(:,1))';
             TR = obj.Prot.VFAData.Mat(:,2);
-            if (length(unique(TR))~=1), error('VFA data must have same TR'); end
-            if ~isfield(data, 'B1map'), data.B1map = []; end
-            if ~isfield(data, 'Mask'), data.Mask = []; end
-            [FitResult.T1, FitResult.M0] = Compute_M0_T1_OnSPGR(double(data.VFAData), flipAngles, TR(1), data.B1map, data.Mask);
+            if obj.voxelwise == 0
+                if (length(unique(TR))~=1), error('VFA data must have same TR'); end
+                if ~isfield(data, 'B1map'), data.B1map = []; end
+                if ~isfield(data, 'Mask'), data.Mask = []; end
+                [FitResult.T1, FitResult.M0] = Compute_M0_T1_OnSPGR(double(data.VFAData), flipAngles, TR(1), data.B1map, data.Mask);
+            elseif obj.voxelwise == 1
```

Comment:
Looks good to me. 

We can implement a similar approach to enable switching between linear-non-voxelwise fitting or nonlin-voxelwise fitting.